### PR TITLE
Update readme, fix grammar

### PR DESCRIPTION
**Pull request by alicebartlett**
- Fix grammar "mostly used" -> "most used"
- Change a whitespace character that's breaking markdown
- Document the babelRuntime flag

@AlbertoElias are you happy of this discription of what babelRuntime=false does?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ All the tasks are built using [gulp](http://gulpjs.com/), and almost all of them
 	   [--scssLintPath=<path>]      Custom scss-lint configuration
 	   [--esLintPath=<path>]        Custom esLint configuration
 	   [--editorconfigPath=<path>]  Custom .editorconfig
+	   [--babelRuntime=false]       Exclude Babel polyfills
```
Comment (AlbertoElias) [inline README.md]: This should go in the build task section of the readme. Currently it's not a common option, so I´m not sure it should go here, but if you think we need to highlight it more, then sure, but you also need to update the cli.js file

**Comment by alicebartlett** [comment]
thanks for the feedback @AlbertoElias, i've updated the pr

File: README.md [hunk 2]
```diff
@@ -101,6 +101,7 @@ Runs:
 	- sourcemaps: `Boolean | 'inline'` Set to true to output sourcemaps as a separate file, even if env is 'production'. Set to 'inline' to output sourcemaps inline (Default: false in production, true in development)
 	- loaders: `Array` Additional Webpack loaders for JavaScript files to run *before* imports-loader (removes AMD module support), babel-loader (which adds babel-runtime polyfills) and textrequireify-loader. OBT will search for loaders in its `node_modules` directory, but also in the project's `node_modules` folder. This way, you can install your own loaders and pass them to the `loaders` array by their name. e.g. `[coffee-loader]`
 	- standalone: `String` Export built file to a global variable with the name passed to this (Default: '')
+	- babelRuntime: `Boolean` Set to true by default. Setting to false will exclude Babel polyfills from the built file
```
Comment (AlbertoElias) [inline README.md]: the structure for defaults is to mention it at the end `(Default: true)`

File: README.md [hunk 3]
```diff
@@ -101,7 +101,7 @@ Runs:
 	- sourcemaps: `Boolean | 'inline'` Set to true to output sourcemaps as a separate file, even if env is 'production'. Set to 'inline' to output sourcemaps inline (Default: false in production, true in development)
 	- loaders: `Array` Additional Webpack loaders for JavaScript files to run *before* imports-loader (removes AMD module support), babel-loader (which adds babel-runtime polyfills) and textrequireify-loader. OBT will search for loaders in its `node_modules` directory, but also in the project's `node_modules` folder. This way, you can install your own loaders and pass them to the `loaders` array by their name. e.g. `[coffee-loader]`
 	- standalone: `String` Export built file to a global variable with the name passed to this (Default: '')
-	- babelRuntime: `Boolean` Set to true by default. Setting to false will exclude Babel polyfills from the built file
+	- babelRuntime: `Boolean` Setting to false will exclude Babel polyfills from the built file. Set to true by default.
```
Comment (AlbertoElias) [inline README.md]: It should be structured as `(Default: true)` like the other options

**Pull request merged**